cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/AbstractTransformerConversion.java

```diff
@@ -120,4 +122,81 @@ protected void addAliases(Identifiable<?> identifiable) {
             }
         }
     }
+
+    private static RegulatingControl getRegulatingControl(Context context, String regulatingControlId) {
+        Objects.requireNonNull(regulatingControlId);
```

Comment:
This method is private. If we are sure the regulatingControlId is not null when the method is called, it is not useful for me to check it.

Reply:
Done